app/themes/user-admin/src/app/header/header.component.html

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-inverse"> 
+<nav class="navbar navbar-inverse square-corners"> 
```

Comment:
As this is a Theme change, we cannot use a class to apply the 'squaring' styling. We need to ID all the elements and over-ride them in our theme CSS instead, as this CSS class will have no meaning if we change themes.